src/components/date-picker/__tests__/DatePicker.test.tsx

```diff
@@ -304,6 +304,22 @@ describe('components/date-picker/DatePicker', () => {
             expect(stopPropagationSpy).not.toHaveBeenCalled();
         });
 
+        test('should not stop propagation when isKeyboardInputAllowed is enabled', () => {
+            const wrapper = renderDatePicker({ isKeyboardInputAllowed: true });
+            const instance = wrapper.instance();
+            const inputEl = wrapper.find('input').at(0);
+            const stopPropagationSpy = jest.fn();
+            if (instance.datePicker) {
```

Comment:
since this is an static wrapper this conditional will always evaluate to true... am I right?

Reply:
This is needed due to a typescript error that pops up when its not there: `Object is possibly 'null'`